xblock/fields.py

```diff
@@ -652,6 +659,12 @@ def is_set_on(self, xblock):
         # pylint: disable=protected-access
         return self._is_dirty(xblock) or xblock._field_data.has(xblock, self.name)
 
+    def none_to_xml(self):
```

Comment:
Maybe just make this a class attribute to start with, and then if a field needs it to be instance specific they can override w/ a property?